scripts/dotnet-cli-build/CompileTargets.cs

```diff
@@ -657,7 +655,6 @@ private static BuildTargetResult CompileCliSdk(BuildTargetContext c, DotNetCli d
                 {
                     // Yes, it is .exe even on Linux. This is the managed exe we're working with
                     File.Copy(Path.Combine(binaryToCorehostifyOutDir, $"{binaryToCorehostify}.exe"), Path.Combine(outputDir, $"{binaryToCorehostify}.dll"));
-                    File.Delete(Path.Combine(binaryToCorehostifyOutDir, $"{binaryToCorehostify}.exe"));
```

Comment:
If an assembly specified by the deps file is not found, I am flagging an error.

So I had to remove this deletion. The other option is to remove this entry from the deps file as well. At that point, it is editing a project.lock.json. Both of these are not pretty, do you have other ideas or opinions? @piotrpMSFT.